packages/adapter-cloudflare-workers/index.js

```diff
@@ -4,6 +4,7 @@ import { execSync } from 'node:child_process';
 import esbuild from 'esbuild';
 import toml from '@iarna/toml';
 import { fileURLToPath } from 'node:url';
+import { getBindingsProxy } from 'wrangler';
```

Comment:
```suggestion
import { getPlatformProxy } from 'wrangler';
```
getBindingProxy has been renamed to getPlatformProxy in wrangler v3.28.2
https://github.com/cloudflare/workers-sdk/pull/5002
https://github.com/cloudflare/workers-sdk/releases/tag/wrangler%403.28.2